test/src/e2e_vm_tests/test_programs/should_pass/language/intrinsics/dbg-release/Forc.lock

```diff
@@ -0,0 +1,8 @@
+[[package]]
+name = "dbg-release"
```

Comment:
Test names are following the `snake_case` convention. To stick to it and make the test same as other, it should be `dbg_release`.